frontend/app/components/nav-bar.js

```diff
@@ -26,4 +31,17 @@ export default class NavBarComponent extends Component {
   toggleTeamCreating() {
     this.isNewTeam = !this.isNewTeam;
   }
+
+  @action
+  searchByQuery() {
+    if (this.searchQuery.trim(' ').length > 2){
+      this.router
+        .transitionTo("teams.index", {
+          queryParams: { q: this.searchQuery, team_id: undefined, folder_id: undefined }
```

Comment:
Again. Change this to null if possible

Reply:
null doesn't work. It just set's the query team_id: ''. null is the content in the string.